core/tox.ini

```diff
@@ -4,7 +4,7 @@ envlist =
 
 [testing]
 deps =
-    grpcio
+    grpcio >= 1.0.2rc0
     mock
```

Comment:
I am :-1: on upper bounds in **testing**. It's good to have a canary **somewhere** that tells us new versions break us.